app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> None:
+
+    completed_runs = {}
+
+    def inner(*args) -> str:
+        if args in completed_runs:
+            print("Getting from cache")
+            return completed_runs[args]
```

Comment:
Move it after if-else
https://github.com/mate-academy/py-cache-decorator/blob/master/checklist.md#dont-repeat-yourself